examples/permissioned-token/assets/permissions.py

```diff
@@ -0,0 +1,156 @@
+from pyteal import *
+
+def approval_program():
+    """
+    Th Permissions smart contract defines transfer rules.
+    Here, we implement two rules:
+        a) receiver asset balance should be <=100 after transfer
+        b) both [from, to] accounts must be whitelisted
+    The Permissions smart contract must be called with the Controller to validate
+    transfers. The Controller smart contract defines a manager account, which
+    is used here to update user attributes and smart contract state.
+    """
+
+    true = Int(1)
+    max_tokens = Bytes("max_tokens") # maximum token transfer rule
+    whitelist_count = Bytes("whitelist_count") # whitelisted accounts counter
+    controller_app_id = Bytes("controller_app_id") # token's controller application index
+
+    # Always verify that the RekeyTo property of any transaction is set to the ZeroAddress
+    # unless the contract is specifically involved ina rekeying operation.
+    rekey_check = Txn.rekey_to() == Global.zero_address()
+
+    """
+    Handles Permissions SSC deployment. Expectes 1 argument
+    * controller_app_id: controller application index
+    During deployment
+    * max_tokens is set to 100
+    * whitelist_counter is intialized to 0
+    * controller application index is saved in global state
+    """
```

Comment:
as above, we should use the `#` comment for things which are not function or class doc string.

Reply:
yes, noted.